docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -235,71 +241,55 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
 
 [NOTE]
 ====
-Your decision to choose whether to use opaque or JWT token format will be driven by architectural considerations.
+Architectural considerations drive your decision to use opaque or JWT token format.
 Opaque tokens tend to be much shorter than JWT tokens but need most of the token-associated state to be maintained in the provider database.
 Opaque tokens are effectively database pointers.
-JWT tokens are significantly longer than the opaque tokens but the providers are effectively delegating storing most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
+JWT tokens are significantly longer than opaque tokens. Still, the providers effectively delegate most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
 ====
 
-The following table provides a summary of the options for each authentication mechanism:
+.Credential types and authentication mechanisms
```

Comment:
This table is a higher level table which should not be placed here in the section comparing OIDC/JWT/OAuth2, 
I'd consider placing at the very top of the doc just above Built in mechanisms. Or may be in the main security concept

Reply:
Moving it to the top for now. Will fix the rest and let you know. Thank you!